modules/social_features/social_landing_page/templates/profile--featured--sky.html.twig

```diff
@@ -0,0 +1,21 @@
+{% extends 'profile--featured.html.twig' %}
```

Comment:
I think we should keep `--sky` templates in the `socialblue` theme because they're a special case of that theme and not part of this module.

Reply:
Fixed.